lib/install.ps1

```diff
@@ -34,9 +34,14 @@ function install_app($app, $architecture, $global, $suggested, $use_cache = $tru
         return
     }
 
-    if ((get_config 'manifest-review' $false) -and ($MyInvocation.ScriptName -notlike '*scoop-update*')) {
-        Write-Output "Manifest: $app.json"
-        Write-Output $manifest | ConvertToPrettyJson
+    if ((get_config 'manifest_review' $false) -and ($MyInvocation.ScriptName -notlike '*scoop-update*')) {
+        "Manifest: $app.json"
+        $style = get_config cat_style ''
+        if (![String]::IsNullOrWhiteSpace($style)) {
```

Comment:
```suggestion
        if (get_config 'cat_style') {
```

![image](https://user-images.githubusercontent.com/5832170/154518430-7d11b8a7-2a0e-4ed6-a3ce-130ac723f17a.png)

Reply:
$style variable needs to be defined. It is used in the pipeline

Reply:
Okay, I'll edit the commit.